docs/en/latest/plugins/ai-rate-limiting.md

```diff
@@ -413,9 +413,9 @@ X-AI-RateLimit-Reset-deepseek-instance: 0
 
 ### Configure Instance Priority and Rate Limiting
 
-The following example demonstrates how you can configure two models with different priorities and apply rate limiting on the instance with a higher priority. In the case where `fallback_strategy` is set to `instance_health_and_rate_limiting`, the Plugin should continue to forward requests to the low priority instance once the high priority instance's rate limiting quota is fully consumed.
+The following example demonstrates how you can configure two models with different priorities and apply rate limiting on the instance with a higher priority. In the case where `fallback_strategy` is set to `["rate_limiting"]`, the Plugin should continue to forward requests to the low priority instance once the high priority instance's rate limiting quota is fully consumed.
```

Comment:
1. Why is the example changed to use the array format? The string is also supported.
2. The same example is also in the `ai-proxy-multi` doc. If it needs to be changed, both should have the same change?

Reply:
okay i will change the other doc as well.

Reply:
Sounds good. Just a reminder that there's more than one example to update.